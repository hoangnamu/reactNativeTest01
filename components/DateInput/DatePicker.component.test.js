import React from "react";
import DatePicker from "./DatePicker.component";
import TestRenderer from "react-test-renderer";
import { shallow, render, mount, configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

configure({ adapter: new Adapter() });

// Make Enzyme functions available in all test files without importing
global.shallow = shallow;
global.render = render;
global.mount = mount;

describe("DatePicker Test ", () => {
  it("Should render snapshot for date picker", () => {
    // RENDER SNAPSHOT FOR DATE PICKER
    const datePickerSnap = TestRenderer.create(<DatePicker />).toJSON()
    expect(datePickerSnap).toMatchSnapshot();
  });
});
