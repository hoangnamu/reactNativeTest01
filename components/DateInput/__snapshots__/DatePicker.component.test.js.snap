// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DatePicker Test  Should render snapshot for date picker 1`] = `
<div>
  <h1>
    "Test"
  </h1>
</div>
`;
